### PR TITLE
BaSP-TG-25: List/Delete Projects

### PR DESCRIPTION
**Pull request by LuliBianchini**
### WORK DONE

- Create table to show list of Projects
- Delete feature
- Add/Edit buttons to redirect to de Create and Edit projects features
- Success modal when delete projects

### CODE REVIEW

File: src/Components/Projects/ListItem/ListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    try {
+      await fetch(`${process.env.REACT_APP_API_URL}api/projects/${_id}`, {
+        method: 'DELETE'
+      });
+      setShowModal(true);
+      setTitleModal('Project deleted');
+    } catch (error) {
+      setShowModal(true);
+      setTitleModal('Error. Project could not be deleted');
+      console.error(error);
```
Comment (agusting19) [inline src/Components/Projects/ListItem/ListItem.jsx:16]: console.log

File: src/Components/Projects/index.js [hunk 2]
```diff
@@ -1,11 +1,32 @@
+import { useEffect, useState } from 'react';
+import List from './List/List';
 import styles from './projects.module.css';
 
-function Projects() {
+const Projects = () => {
+  const [list, setProjects] = useState([]);
```
Comment (agusting19) [inline src/Components/Projects/index.js:6]: "list" can be "projects" to be more readable

**Review by agusting19 — DISMISSED** [review]
You have to make little fixes

**Comment by tinchorova** [comment]
## Testing results
Tested locally on https://alexis-trackgenix-app-git-feature-tg-25-list-60c4ba-basp-m2022.vercel.app/projects

**Test Results: Failed** ⚠️ 

Bugs found:

1. There should be a confirmation/cancel window when clicking a delete project button, instead it automatically deletes the project 
image

**Review by tinchorova — CHANGES_REQUESTED** [review]
Please refer to https://github.com/BaSP-m2022/alexis-trackgenix-app/pull/3#issuecomment-1145381158

**Comment by tinchorova** [comment]
Testing Results

Tested locally on https://alexis-trackgenix-app-git-feature-tg-27-list-78d848-basp-m2022.vercel.app/time-sheets

Test Results: Failed ⚠️

Bugs found:

1.When deleting a project, the delete modal is fixed and the user can interact with the rest of the page
image
-------------
image

**Review by tinchorova — CHANGES_REQUESTED** [review]
Please refer to: https://github.com/BaSP-m2022/alexis-trackgenix-app/pull/3#issuecomment-1146266938

**Comment by tinchorova** [comment]
Testing Results

Tested locally on https://alexis-trackgenix-app-git-feature-tg-27-list-78d848-basp-m2022.vercel.app/time-sheets

Test Results: Failed ⚠️

Bugs found:
1. The problem was not fixed, user can still scroll up and down when modal is shown.

**Review by tinchorova — APPROVED** [review]
Testing Results

Tested locally on https://alexis-trackgenix-app-git-feature-tg-25-list-60c4ba-basp-m2022.vercel.app/projects
Test result: Approved!

**Keep in mid: User shouldn't be allowed to scroll while a modal is shown**

**Review by iaracriscenti — APPROVED** [review]

**Pull request merged**